syntaxes/gdb.tmLanguage.json

```diff
@@ -33,12 +33,61 @@
 								"2": { "name": "support.other.regexp.gdb" }
 							}
 						},
+						{
+							"match": "(-function)\\s+([^\\s]+)",
+							"captures": {
+								"1": { "name": "support.function.option.gdb" },
+								"2": { "name": "markup.underline.link.location.gdb" }
+							}
+						},
 						{
 							"match": "(-fi(?:le)?)\\s+([^\\s]+)",
 							"captures": {
 								"1": { "name": "support.function.option.gdb" },
 								"2": { "name": "markup.underline.link.file.gdb" }
 							}
+						},
+						{
+							"match": "(-gf(?:ile)?)\\s+([^\\s]+)",
+							"captures": {
+								"1": { "name": "support.function.option.gdb" },
+								"2": { "name": "support.other.regexp.gdb" }
+							}
+						},
+						{
+							"match": "(function)\\s+([^\\s]+)",
+							"captures": {
+								"1": { "name": "keyword.control.command.gdb" },
+								"2": { "name": "markup.underline.link.location.gdb" }
+							}
+						},
+						{
+							"match": "(fi(?:le)?)\\s+([^\\s]+)",
+							"captures": {
+								"1": { "name": "keyword.control.command.gdb" },
+								"2": { "name": "markup.underline.link.file.gdb" }
+							}
+						},
+						{
+							"match": "(d(?:el(?:e(?:t(?:e)?)?)?)?)(\\s+\\d+)*",
+							"captures": {
+								"1": { "name": "keyword.control.command.gdb" },
+								"2": { "name": "entity.name.tag.breakpoint.gdb" }
+							}
+						},
+						{
+							"match": "(en(?:a(?:b(?:l(?:e)?)?)?)?(\\s+\\d+)*",
+							"captures": {
+								"1": { "name": "keyword.control.command.gdb" },
+								"2": { "name": "entity.name.tag.breakpoint.gdb" }
+							}
+						},
+						{
+							"match": "(dis(?:a(?:b(?:l(?:e)?)?)?)?(?:\\s+\\d+)*",
```

Comment:
`skip di` is valid for disable, proposing `"(di(?:s(?:a(?:b(?:l(?:e)?)?)?)?)?(?:\\s+\\d+)*"`

Reply:
taken